ement-lib.el

```diff
@@ -597,12 +597,12 @@ Returns one of nil (meaning default rules are used), `all-loud',
                                          (equal "room_id" key)
                                          (equal (ement-room-id room) pattern)))))
                 (mute-rule-p
-                 (rule) (and (= 1 (length (alist-get 'actions rule)))
-                             (equal "dont_notify" (elt (alist-get 'actions rule) 0))))
+                 (rule) (when-let ((actions (alist-get 'actions rule)))
+                          (seq-contains-p actions "dont_notify")))
```

Comment:
I'm preserving the current logic here but "dont_notify" should be ignored by spec. Instead, ement is supposed to notify iff the "notify" action is present.

See the note in https://spec.matrix.org/v1.7/client-server-api/#actions.